pkg/policy/repository.go

```diff
@@ -39,7 +39,9 @@ type Repository struct {
 
 // NewPolicyRepository allocates a new policy repository
 func NewPolicyRepository() *Repository {
-	return &Repository{}
+	return &Repository{
+		revision: 1,
```

Comment:
Add a comment indicating why we are starting at revision one.

Reply:
Done.